changelogs/8.7.asciidoc

```diff
@@ -86,4 +87,4 @@ https://github.com/elastic/apm-server/compare/v8.7.0\...v8.7.1[View commits]
 - Add overflow metrics collection {pull}10330[10330]
 - Improve APM UI query performance by adding an always enabled service summary metrics aggregator which produces `1m`, `10m` and `60m` aggregation intervals. Store them into data streams `metrics-apm.service_summary.${interval}` and make them hidden for
 non-default intervals {pull}10029[10029] {pull}10060[10060] {pull}10070[10070]
-- Add `service.language.name` {pull}10111[10111]
\ No newline at end of file
+- Add `service.language.name` {pull}10111[10111]
```

Comment:
What changed on this line? In the diff they look kind of identical

Reply:
double dash

Reply:
this is vim fixing the file by automatically adding an EOL (see https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline)

Reply:
oh I confused the `-` diff for a dash 🤦🏼